### PR TITLE
Add license information to docs

### PR DESCRIPTION
**Pull request by SebastianSchildt**
This adds some more verbose license information to the documentation.

I would consider this a FAQ https://github.com/COVESA/vehicle_signal_specification/wiki/VSS-FAQ---Collect-Questions-and-Answers , so maybe since agreed upon this information is also useful elsewhere.

### CODE REVIEW

**Comment by paulboyes** [comment]
Suggested disclaimer:

DISCLAIMER: The following information about open source licensing shall not be considered legal advice. For advice on licensing, COVESA suggests that you contact your open source administrator or legal counsel.

**Comment by SebastianSchildt** [comment]
Added disclaimer

image

File: docs-gen/content/license/_index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+chapter: false
+---
+
+
+{{% notice warning %}}
+**DISCLAIMER:** The following information about open source licensing shall not be considered legal advice. For advice on licensing, COVESA suggests that you contact your open source officer or legal counsel.
+{{% /notice %}}
+## Which license applies
+
+VSS as well as the VSS-tools are covered by the [Mozilla Public License 2.0 (MPL-2.0)](https://opensource.org/licenses/MPL-2.0). The MPL-2.0 is an OSI approved Open Source license that gives you a lot of freedom using VSS in your products, or for (academic) research while providing a solid legal framework should you choose to contribute to either the standards or the tools.
```
Comment (erikbosch) [inline docs-gen/content/license/_index.md:13]: I am not a native english speaker, but for me the last sentence becomes a bit complex. Should we possibly benefit from splitting into two sentences, possibly like below?

Current:

`The MPL-2.0 is an OSI approved Open Source license that gives you a lot of freedom using VSS in your products, or for (academic) research while providing a solid legal framework should you choose to contribute to either the standards or the tools.`

Possible split:

`The MPL-2.0 is an OSI approved Open Source license that gives you a lot of freedom using VSS in your products or for (academic) research. It also provides a solid legal framework if you choose to contribute to either the VSS standard or the tools.`

Comment (erikbosch) [inline docs-gen/content/license/_index.md:13]: I am not a native english speaker, but for me the last sentence becomes a bit complex. Should we possibly benefit from splitting into two sentences, possibly like below?

Current:

`The MPL-2.0 is an OSI approved Open Source license that gives you a lot of freedom using VSS in your products, or for (academic) research while providing a solid legal framework should you choose to contribute to either the standards or the tools.`

Possible split:

`The MPL-2.0 is an OSI approved Open Source license that gives you a lot of freedom using VSS in your products or for (academic) research. It also provides a solid legal framework if you choose to contribute to either the VSS standard or the tools.`

File: docs-gen/content/license/_index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+We will line out some answers to commonly asked question, but keep in mind this page is informational. The legally binding clauses can be found in the [license](https://opensource.org/licenses/MPL-2.0) itself. If in doubt, consult your friendly neighborhood lawyer or IP department.
```
Comment (erikbosch) [inline docs-gen/content/license/_index.md:15]: `commonly asked question` -> `commonly asked questions`?

Comment (erikbosch) [inline docs-gen/content/license/_index.md:15]: `commonly asked question` -> `commonly asked questions`?

File: docs-gen/content/license/_index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Can I use VSS / VSS-tools in my commercial products
+Most definitely yes. You must inform your users that the product contains MPLed code and need to provide access to the source (e.g. by linking to this documentation or our[our Github repository](https://github.com/covesa/vehicle_signal_specification)).
```
Comment (erikbosch) [inline docs-gen/content/license/_index.md:18]: One `our` to much

Comment (erikbosch) [inline docs-gen/content/license/_index.md:18]: One `our` to much

**Review by erikbosch — APPROVED** [review]
Some minor editorial comments

File: docs-gen/content/license/_index.md (hunk 1 above)
Comment (SebastianSchildt) [reply docs-gen/content/license/_index.md]: Done

File: docs-gen/content/license/_index.md (hunk 2 above)
Comment (SebastianSchildt) [reply docs-gen/content/license/_index.md]: 👍

File: docs-gen/content/license/_index.md (hunk 3 above)
Comment (SebastianSchildt) [reply docs-gen/content/license/_index.md]: Done.

**Review by erikbosch — APPROVED** [review]

**Comment by danielwilms** [comment]
@SebastianSchildt fixed typos according to @erikbosch. Can be merged.

**Pull request merged**